sqsdomain/routable_pool.go

```diff
@@ -1,41 +0,0 @@
-package sqsdomain
-
-import (
-	"context"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
-
-	"github.com/osmosis-labs/osmosis/osmomath"
-	poolmanagertypes "github.com/osmosis-labs/osmosis/v25/x/poolmanager/types"
-)
-
-type RoutablePool interface {
```

Comment:
Note to reviewer: this was simply moved to `domain` package due to import cycle issues.